scannerazure.properties

```diff
@@ -6,8 +6,18 @@ organizationUrl=https://www.sonarsource.com
 homepageUrl=https://docs.sonarqube.org/latest/analysis/scan/sonarscanner-for-azure-devops/
 issueTrackerUrl=https://sonarsource.atlassian.net/jira/software/c/projects/VSTS/issues
 sourcesUrl=https://github.com/SonarSource/sonar-scanner-vsts
-archivedVersions=4.7,4.7.1,4.7.2,4.8,4.8.1,4.9,4.10,4.11,4.12,4.16,4.17,4.18,4.19,4.20,4.21,4.22,4.23,4.23.1,5.0.0,5.1.0,5.1.1,5.2.0,5.3.0,5.4.0,5.5.0,5.6.0,5.6.1,5.7.0,5.8.0,5.8.1,5.9.0,5.10.0,5.11.0,5.11.1,5.12.0,5.13.0,5.14.0
-publicVersions=5.15.0
+archivedVersions=4.7,4.7.1,4.7.2,4.8,4.8.1,4.9,4.10,4.11,4.12,4.16,4.17,4.18,4.19,4.20,4.21,4.22,4.23,4.23.1,5.0.0,5.1.0,5.1.1,5.2.0,5.3.0,5.4.0,5.5.0,5.6.0,5.6.1,5.7.0,5.8.0,5.8.1,5.9.0,5.10.0,5.11.0,5.11.1,5.12.0,5.13.0,5.14.0,5.15.0,5.16.0
+publicVersions=5.17.0
```

Comment:
The latest SQ extension version is [5.17.2](https://github.com/SonarSource/sonar-scanner-vsts/blob/master/extensions/sonarqube/vss-extension.json#L5C15-L5C21).

After `5.17.0,` , 2 patches 5.17.1 and 5.17.2. `5.17.2` was released. `5.17.2` is the current latest version which works.

Reply:
good catch, thanks. I updated to 5.17.2